testing/runner.ts

```diff
@@ -96,7 +97,16 @@ export class Runner {
               console.error(`\n    Actual:\n`);
               console.error(indent(JSON.stringify(result.err.actual, null, 4), 8));
             }
-            console.error("\n");
+            console.error(`\n    Overall diff:`);
+            Diff.diffLines(
+              indent(JSON.stringify(result.err.expected)),
```

Comment:
I think you probably want to indent these after?

Reply:
Meh, either works